subsys/net/lib/nrf_cloud/src/nrf_cloud_codec.c

```diff
@@ -889,20 +859,283 @@ static int nrf_cloud_encode_service_info_ui(const struct nrf_cloud_svc_info_ui *
 	return 0;
 }
 
+#ifdef CONFIG_MODEM_INFO
+
+#define MODEM_INFO_JSON_KEY_NET_INF	"networkInfo"
+#define MODEM_INFO_JSON_KEY_SIM_INF	"simInfo"
+#define MODEM_INFO_JSON_KEY_DEV_INF	"deviceInfo"
```

Comment:
These should probably be defined in the `.h` file, and not be gated behind CONFIG_MODEM_INFO, since they are conformal to the schema laid out here:
https://github.com/nRFCloud/application-protocols/blob/v1/schemas/deviceToCloud/device/device.json

(See `properties->data->properties`)

I would also rename them to match with the naming convention used by the other properties in the `.h` file

Reply:
fixed